### ISSUE TITLE
IsOpen always return false 

### ISSUE THREAD

**Issue by IlliaPruskii** [ammarahm-ed/react-native-actions-sheet#215]
Hello! 
I need to detect which drawer is open right now, but isOpen function always returns false. You can find my code below, thanks!
```jsx
import { StyleSheet, View, Button } from 'react-native';
import ActionSheet, { SheetProvider, registerSheet, SheetManager, SheetProps } from "react-native-actions-sheet";

const EXAMPLE_SHEET = 'EXAMPLE_SHEET';

function ExampleSheet(props: SheetProps) {

  return (
    <ActionSheet id={props.sheetId}>
      <View>
        <Button title='IsOpen' onPress={() => console.log('isOpenResult', SheetManager.get(EXAMPLE_SHEET).current.isOpen())}/>
      </View>
    </ActionSheet>
  );
}

registerSheet(EXAMPLE_SHEET, ExampleSheet);

export default function App() {
  return (
    <SheetProvider>
      <View style={styles.container}>
        <Button title='Open drawer' onPress={() => SheetManager.show(EXAMPLE_SHEET)}/>
      </View>
    </SheetProvider>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
});
```
Note: version 0.8.4 and I used it with expo

**Comment by ammarahm-ed** [comment]
Fixed in v0.8.7

**Comment by IlliaPruskii** [comment]
Thanks!
